server.js

```diff
@@ -76,6 +77,12 @@ app.get('/config.js', (req, res) => {
   }
 });
 
+// new metrics endpoint for Walkthrough applciation.
```

Comment:
Minor thing, but maybe a more timeless comment like `// prometheus metrics endpoint` would be better here